alacritty/src/renderer/platform.rs

```diff
@@ -21,15 +21,25 @@ use winit::platform::x11;
 pub fn create_gl_display(
     raw_display_handle: RawDisplayHandle,
     _raw_window_handle: Option<RawWindowHandle>,
+    #[cfg(any(windows, all(feature = "x11", not(any(target_os = "macos", windows)))))]
+    prefer_egl: bool,
```

Comment:
Would be easier to just add this on all platforms and avoid the cfg on the caller.

Reply:
So we're okay with passing unused parameters? I was a bit hesitant about this since it's only a bool, and I can agree with you in this case.

Reply:
yeah, for debug config options it's fine to do so.